src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsServer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is there a reason for changing the copyright year here? I don't see any other change in this file.

Reply:
reverted mistake.